assets/armsnippets.jsonc

```diff
@@ -140,6 +140,120 @@
         ],
         "description": "Application Service Plan (Server Farm)"
     },
+    "Application Gateway": {
+        "prefix": "arm-app-gw",
+        "body": [
+            "{",
+            "    \"name\": \"${1:applicationGatewayName}\",",
+            "    \"type\": \"Microsoft.Network/applicationGateways\",",
+            "    \"apiVersion\": \"2019-11-01\",",
+            "    \"location\": \"[resourceGroup().location]\",",
+            "    \"dependsOn\": [",
+            "        \"[resourceId('Microsoft.Network/virtualNetworks', '${2:vnetName}')]\",",

```

Comment:
Change name to `virtualNetwork1` to align with VNET snippet.

![image](https://user-images.githubusercontent.com/7844635/77958930-ac4cf780-728a-11ea-9453-f66381d7445d.png)